client/blindbox/requests/secure_session.py

```diff
@@ -11,12 +11,14 @@ class SecureSession(Session):
     def __init__(
         self,
         addr: str,
+        cce_file: str,
         debug_mode: bool = False,
     ):
         """Connect to a BlindBox service.
         Please refer to the connect function for documentation.
         Args:
             addr (str):
+            cce_file (str):
```

Comment:
It would have been nice to document a bit here the process of attestation validation.